### PR TITLE
Add completed and started signals to Action

### PR DESCRIPTION
**Pull request by larryonoff**
**What's in this pull request?**

This PR adds `completed` and `started` signals to `Action`. I often use empty `Action` as a trigger that some logic should begin executing. So these two properties were introduced.

### CODE REVIEW

File: Source/Action.swift [hunk 1]
```diff
@@ -14,6 +14,26 @@ extension Action {
     public static var rex_disabled: Action {
         return Action(enabledIf: ConstantProperty(false)) { _ in .empty }
     }
+
+    /// Whether the action execution was completed succesfully.
```
Comment (dmcrodrigues) [inline Source/Action.swift]: You have a typo here.

`succesfully` → `successfully`

Comment (larryonoff) [inline Source/Action.swift]: Thank you! Fixed it.

**Comment by RuiAAPeres** [comment]
@larryonoff I really like this! Could you add some tests as well?

**Comment by larryonoff** [comment]
Sure. Will do it a bit later.

**Comment by larryonoff** [comment]
@RuiAAPeres Just added tests.

File: Tests/ActionTests.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        action
+            .apply()
+            .start()
+
+        XCTAssertTrue(started)
+    }
+    
+    func testCompleted() {
+        let (producer, sink) = SignalProducer<Int, TestError>.buffer(Int.max)
+        let action = Action { producer }
```
Comment (RuiAAPeres) [inline Tests/ActionTests.swift:37]: could you change `sink` to `observer`. The `sink` nomenclature was abandoned a while back. (reactivecocoa/reactivecocoa#2475)

Comment (larryonoff) [inline Tests/ActionTests.swift:37]: Sure. Will do it in 2 hours.

Comment (larryonoff) [inline Tests/ActionTests.swift:37]: Done.

**Comment by RuiAAPeres** [comment]
@larryonoff could you rebase?

**Comment by larryonoff** [comment]
@RuiAAPeres rebase done.

**Pull request merged**